utils/variant_qc_plots.py

```diff
@@ -1,5 +1,5 @@
 from gnomad_hail import *
-from gnomad_hail.resources.variant_qc import *
+from gnomad_qc.resources.variant_qc import *
```

Comment:
Same here, please add a `#FIXME` so we know this is something that needs to be tackled.